docs/maya/muscle.md

```diff
@@ -63,6 +64,14 @@ To create an AdnMuscle, follow these steps:
 | **Use Custom Stiffness** | Boolean | False          | ✓ | Toggles the use of a custom stiffness value. If custom stiffness is used, *Material* and *Stiffness Multiplier* will be disabled and *Stiffness* will be used instead. |
 | **Stiffness**            | Float   | 10<sup>5</sup> | ✓ | Sets the custom stiffness value. Its value must be greater than 0.0. |
 
+#### Mass Properties
+
+| Name | Type | Default | Animatable | Description |
+| :--- | :--- | :------ | :--------- | :---------- |
+| **Point Mass Mode**        | Enumerator | By Uniform Value | ✓ | Defines how masses should be used in the solver.<ul><li>*By Density* allows to estimate the mass value by multiplying Density * Volume.</li><li>*By Uniform Value* allows to set a uniform mass value.</li></ul> |
+| **Density**                | Float      | 1060.0           | ✓ | Sets the density value in kg/m3 to be able to estime mass values with *By Density* mode. Has a range of \[0.001, 10<sup>6</sup>\]. Lower and upper limits are soft, lower and higher values can be used. |
```

Comment:
For the definition of the density attribute (in all deformers) I wrote "Sets the density value in kg/m3..", but I'm not sure if we should specify scene units instead of meters.

Reply:
I believe we should write it in scene units. My suggestions for this would be one of these two:

Directly write it down
```suggestion
| **Density**                | Float      | 1060.0           | ✓ | Sets the density value in kilograms per cubic scene units to be able to estime mass values with *By Density* mode. Has a range of \[0.001, 10<sup>6</sup>\]. Lower and upper limits are soft, lower and higher values can be used. |
```

Or write scene units as a symbol
```suggestion
| **Density**                | Float      | 1060.0           | ✓ | Sets the density value in kg/su<sup>3</sup> (cubic scene uinits) to be able to estime mass values with *By Density* mode. Has a range of \[0.001, 10<sup>6</sup>\]. Lower and upper limits are soft, lower and higher values can be used. |
```

Reply:
I would keep it as kg/m3 as it is the exact density parameter that was deduced from the papers. Changing it to kg/u3 will induce the user to think that there is an interna conversion happening. In the actual deformer tooltip we use kg/m3 so I would personally keep it like that to be "physically" correct. But let's have a chat before the release

Reply:
I will keep it as kg/m3, but writing the superscript properly.